Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -126,6 +126,7 @@ bool AVIDump::CreateVideoFile()
   const std::string& s_format = g_Config.sDumpFormat;
 
   std::string s_dump_path = GetDumpPath(s_format);
+  File::CreateFullPath(s_dump_path);
 
   if (s_dump_path.empty())
```

Comment:
Might you move this call after this condition here?

Reply:
I dont think it matters, as s_dump_path can only be empty when the file exists, due to the logic of GetDumpPath.

Reply:
I would prefer to have it afterwards, because then future readers of the code aren't going to wonder as much about whether the placement really is fine, and because the logic in GetDumpPath might change in the future.